muxplore/main.c

```diff
@@ -2311,6 +2346,8 @@ int main(int argc, char *argv[]) {
     pthread_t joystick_thread;
     pthread_create(&joystick_thread, NULL, (void *) joystick_task, NULL);
 
+    init_footer_elements();
```

Comment:
When init_elements was moved in this commit 
https://github.com/MustardOS/frontend/commit/0ec63b8fbe5f1dadff8789ee026f1d415949a770
it caused a bug where not all the glyph elements in the footer where displayed because it was being called before any items had been generated.  Moving it back though breaks how the artwork loading fix so I broke out the portion that just handles the glyph creation into it's own method.